docs/rules/structured-imports.md

```diff
@@ -0,0 +1,33 @@
+# Enforces a certain structure in your import statements
```

Comment:
Avoid vague terms like "certain structure".  Be direct and concise.

`Enforces order of import statement types`.